tests/sdk.test.ts

```diff
@@ -1,94 +1,100 @@
-
 import Web3 from "web3";
-import { ethers } from "ethers";
 import Assets from "../src/assets.json";
-import { ExternalProvider, Web3Provider } from "@ethersproject/providers";
-import { Synths } from "../src";
-import { Asset } from "../src/library/Asset";
+import {ExternalProvider, Web3Provider} from "@ethersproject/providers";
```

Comment:
oops, I prefer the space on either side. Looks like my ide thought that was waste. I'll change this back.